main.py

```diff
@@ -83,13 +84,16 @@ def extract(entity_json: str) -> Dict[str, Any]:
         response: File response for Fast API.
     """
     entities = json.loads(entity_json)
-    for name, description in entities.items():
-        entities[name] = Field(description=description)
+
     # Creating a new pydantic object from the entity
-    FormatModel = type("FormatModel", (BaseModel,), entities)
+    format_model = create_model("FormatModel", **entities)
```

Comment:
this seems to create a pydantic object with the entity descriptions as default values rather than description. We can do a quick test on company to verify that this is true.

I think we still need the conversion to `Field(description=description)` to be in line with what langchain uses here - https://python.langchain.com/en/latest/modules/prompts/output_parsers/examples/pydantic.html

Reply:
Just verified and you're right. Fixing